other/dijkstra_bankers_algorithm.py

```diff
@@ -0,0 +1,233 @@
+# A Python implementation of the Banker's Algorithm in Operating Systems using
+# Processes and Resources
+# {
+# "Author: "Biney Kingsley (bluedistro@github.io), bineykingsley36@gmail.com",
+# "Date": 28-10-2018
+# }
+'''
+The Banker's algorithm is a resource allocation and deadlock avoidance algorithm
+developed by Edsger Dijkstra that tests for safety by simulating the allocation of
+predetermined maximum possible amounts of all resources, and then makes a "s-state"
+check to test for possible deadlock conditions for all other pending activities,
+before deciding whether allocation should be allowed to continue.
+[Source] Wikipedia
+[Credit] Rosetta Code C implementation helped very much.
+ (https://rosettacode.org/wiki/Banker%27s_algorithm)
+'''
+
+import numpy as np
+import time
+
+
+class Bankers_algorithm:
```

Comment:
__class BankersAlgorithm:__  # CamelCase on ClassNames as discussed in CONTRIBUTING.md